schemas/json/layout/layout.schema.v1.json

```diff
@@ -556,7 +556,6 @@
       }
     },
     "groupPanelOptions": {
-      "additionalProperties": false,
```

Comment:
Removing this line just makes it possible to type these in without having any schema validation. The root cause of this must be something else, because these properties are defined in `#/definitions/panelComponent` that is referenced in the `allOf` below..